README.md

```diff
@@ -18,7 +18,11 @@ To run a benchmark, use:
 
 ```bash
 pipenv install
-pipenv run ./run.sh
+pipenv run ./run.sh <EQ_ID> <FORM_TYPE>
+```
+e.g.
+```
+pipenv run ./run.sh test mutually_exclusive
 ```
 
 This will run 1 minute of locust requests with 1 user and no wait time between requests. It will also process the CSV output to combine `collection_exercise_sid` into `[ID]` in the requests.
```

Comment:
` It will also process the CSV output to combine `collection_exercise_sid` into `[ID]` in the requests.` is that still correct?